src/gui/widgets/text_box_base.cpp

```diff
@@ -153,8 +157,13 @@ void text_box_base::set_cursor(const std::size_t offset, const bool select)
 		queue_redraw();
 
 	} else {
-		assert(offset <= text_.get_length());
-		selection_start_ = offset;
+		// doesn't work for scroll_text
+//		assert(offset <= text_.get_length());
```

Comment:
I don't think there's a reason to leave this comment in either?

Reply:
Removed.